scripts/Muon/GUI/Common/load_file_widget/model.py

```diff
@@ -0,0 +1,103 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2018 ISIS Rutherford Appleton Laboratory UKRI,
+#     NScD Oak Ridge National Laboratory, European Spallation Source
+#     & Institut Laue - Langevin
+# SPDX - License - Identifier: GPL - 3.0 +
+from __future__ import (absolute_import, division, print_function)
+
+import os
+
+from mantid.kernel import ConfigService
+from mantid.api import WorkspaceGroup
+
+from Muon.GUI.Common.muon_load_data import MuonLoadData
+import Muon.GUI.Common.utilities.load_utils as load_utils
+
+
+def is_workspace_group(workspace):
```

Comment:
this is also in the previous PR (identical). So I think we should make some shared functionality.